### PR TITLE
Add VIA support to Noxary 268

### PR DESCRIPTION
**Pull request by Rozakiin**
## Description

Adds VIA support to the Noxary 268.
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/noxary/268/268.c [hunk 1]
```diff
@@ -1 +1,44 @@
...
+// }
+
+// void led_set_user(uint8_t usb_led) {
+//     if (usb_led & (1 << USB_LED_CAPS_LOCK)) {
+//         DDRB |= (1 << 6); PORTB |= (1 << 6);
+//     }
+//     else {
+//         DDRB &= ~(1 << 6); PORTB &= ~(1 << 6);
+//     }
+// }
```
Comment (fauxpark) [inline keyboards/noxary/268/268.c:44]:
```suggestion
```

File: keyboards/noxary/268/rules.mk [hunk 2]
```diff
@@ -12,15 +12,22 @@ MCU = atmega32u4
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality on B7 by default
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs
```
Comment (fauxpark) [inline keyboards/noxary/268/rules.mk:33]:
```suggestion
```

File: keyboards/noxary/268/rules.mk [hunk 3]
```diff
@@ -12,15 +12,22 @@ MCU = atmega32u4
...
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality on B7 by default
```
Comment (fauxpark) [inline keyboards/noxary/268/rules.mk:26]:
```suggestion
BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality
```

File: keyboards/noxary/268/keymaps/via/readme.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+![Noxary 268 Layout Image](https://i.imgur.com/D9jQz6g.png)
+
+# The VIA keymap for Noxary 268
```
Comment (fauxpark) [inline keyboards/noxary/268/keymaps/via/readme.md:3]:
```suggestion
# The VIA keymap for Noxary 268

![Noxary 268 Layout Image](https://i.imgur.com/D9jQz6g.png)
```

File: keyboards/noxary/268/keymaps/via/rules.mk [hunk 5]
```diff
@@ -0,0 +1,5 @@
+VIA_ENABLE = yes
+MOUSEKEY_ENABLE = no
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+LINK_TIME_OPTIMIZATION_ENABLE = yes
```
Comment (noroadsleft) [inline keyboards/noxary/268/keymaps/via/rules.mk:5]: Same thing but less unwieldy.

```suggestion
LTO_ENABLE = yes
```

File: keyboards/noxary/268/readme.md [hunk 6]
```diff
@@ -6,7 +6,6 @@ A fully customizable 65% keyboard.
 
 * Keyboard Maintainer: [Rozakiin](https://github.com/rozakiin)
 * Hardware Supported: 268 PCB
-  * rev1
 * Hardware Availability: [Noxary](https://www.instagram.com/noxaryco/)
```
Comment (Erovia) [inline keyboards/noxary/268/readme.md:9]:
```suggestion
* Hardware Availability: [Noxary](https://noxary.co)
```
Their instagram is empty(?), but their website seems to be alive.

File: keyboards/noxary/268/keymaps/via/rules.mk (hunk 5 above)
Comment (Erovia) [reply keyboards/noxary/268/keymaps/via/rules.mk:5]: And better known.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**